client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -213,6 +218,10 @@ export const RequestLineEdit = ({
                 label={t('label.amc')}
                 sx={{ marginBottom: 1 }}
               />
+              {line &&
+                plugins.requestRequisitionColumn?.editViewColumns?.map(
+                  (Column, index) => <Column key={index} line={line} />
```

Comment:
I think we should call this `Field`, and `editViewFields` ? `Column` has me thinking I should be defining a column definition with cell/accessor etc..

Reply:
Change here, sorry lots of other changes https://github.com/msupply-foundation/open-msupply/commit/08f318054560f22156a90b5bfdabdbdb81b44a05